### PR TITLE
fix(hero): allow multiple images in hero background

### PR DESCRIPTION
**Pull request by l-bonanni**

### CODE REVIEW

File: components/Heading.vue [hunk 1]
```diff
@@ -3,3 +3,9 @@
     <slot></slot>
   </h3>
 </template>
+
+<style scoped>
+h3 {
+    font-size: 30px;
```
Comment (mfrachet) [inline components/Heading.vue]: Shouldn't we use `em` instead of px?

File: pages/index.vue [hunk 2]
```diff
@@ -33,6 +36,9 @@ import AppArticle from "~/components/Article.vue";
 import AppHero from "~/components/Hero.vue";
 
 export default {
+  data() {
+    return { images: ['/evolution/evolution-left.jpg', '/evolution/evolution-right.jpg'] };
```
Comment (mfrachet) [inline pages/index.vue]: we should maybe rename `images` in `evolution` or `evolutionImages`, WYT?

File: components/Heading.vue (hunk 1 above)
Comment (l-bonanni) [reply components/Heading.vue]: another PR

File: components/Subtitle.vue [hunk 3]
```diff
@@ -4,3 +4,8 @@
   </h2>
 </template>
 
+<style scoped>
+h2 {
+  font-size: 45px;
```
[the remark this answers is not in the document]
Comment (l-bonanni) [reply components/Subtitle.vue]: another PR

**Pull request merged**